parts/k8s/containeraddons/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -626,7 +647,6 @@ metadata:
   name: calico-typha-horizontal-autoscaler
   namespace: kube-system
   labels:
-    kubernetes.io/cluster-service: "true"
```

Comment:
Is removing these lines intended to sync up with the upstream calico manifests?

Reply:
No. I thought this label is deprecated based on https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/README.md . Not sure if it is used by other components other than `addon manager`. WDYT?

Reply:
Based on that description (and the fact that we explicitly use `Reconcile` in these cases), it shouldn't change `addon-manager`'s behavior anyway. I just don't know if the label is used by anything else. I'm going with "probably safe." 😄

Reply:
Cool. That is what I am thinking as well.